src/components/apps/launch/AppLaunchForm.js

```diff
@@ -572,46 +728,21 @@ const AppLaunchForm = (props) => {
                         {isSubmitting ? (
                             <StepperSkeleton baseId={formId} ref={stepperRef} />
                         ) : (
-                            <Stepper
-                                alternativeLabel
-                                nonLinear
+                            <LaunchStepper
+                                steps={steps}
+                                handleStep={handleStep}
+                                handleNext={handleNext}
+                                handleBack={handleBack}
                                 activeStep={activeStep}
+                                isLastStep={isLastStep}
+                                formId={formId}
+                                errors={errors}
+                                touched={touched}
+                                groups={groups}
                                 ref={stepperRef}
-                                className={classes.stepper}
-                            >
-                                {steps.map((step, index) => (
-                                    <Step key={step.label}>
-                                        <StepButton
-                                            id={buildDebugId(
-                                                formId,
-                                                ids.BUTTONS.STEP,
-                                                index + 1
-                                            )}
-                                            onClick={handleStep(index)}
-                                        >
-                                            <StepLabel
-                                                error={
-                                                    !!displayStepError(
-                                                        index,
-                                                        errors,
-                                                        touched,
-                                                        groups
-                                                    )
-                                                }
-                                            >
-                                                <Hidden xsDown>
-                                                    {step.label}
-                                                </Hidden>
-                                            </StepLabel>
-                                        </StepButton>
-                                    </Step>
-                                ))}
-                            </Stepper>
+                            />
                         )}
-                        <Container
-                            component="div"
-                            className={classes.stepContainer}
-                        >
+                        <Box component="div" className={classes.stepContainer}>
```

Comment:
Changing `Container` to `Box` fixed the horizontal scroll issue for most apps in mobile view. This is still an issue with `Tuxedo` app. No visible diff in large screen.